src/components/environment-variables/index.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import usePollEnvVars from './use-poll-env-vars';
+import EnvironmentVariablesList from './environment-variables-list';
+
+const EnvironmentVariables = (props) => {
+  const {
+    appName,
+    envName,
+    componentName,
+    componentType,
+    includeRadixVars,
+    readonly,
+  } = props;
+  const [context, setContext] = useState({ paused: false });
+  const [pollEnvVarsState] = usePollEnvVars(
+    appName,
+    envName,
+    componentName,
+    context
+  );
+  return (
+    <EnvironmentVariablesList
```

Comment:
Handle loading state and errors for pollEnvVarsState. If it fails, no error is displayed